src/ophyd_async/epics/_backend/_aioca.py

```diff
@@ -35,7 +36,59 @@
     dbr.DBR_CHAR: "string",
     dbr.DBR_LONG: "integer",
     dbr.DBR_DOUBLE: "number",
+    dbr.DBR_ENUM: "string",
 }
+_common_meta = {
+    "units",
+    "lower_alarm_limit",
+    "upper_alarm_limit",
+    "lower_ctrl_limit",
+    "upper_ctrl_limit",
+    "lower_disp_limit",
+    "upper_disp_limit",
+    "lower_warning_limit",
+    "upper_warning_limit",
+    "precision",
```

Comment:
@danielballan do we want all these to appear in the descriptor document?

Reply:
I'm happy to turn down to just units, precision. Currently for my downstream purposes I turn lower_ctrl_limit into soft_limit_min for NeXus, so if we can keep just one that's the one I want.

Reply:
Only units and precision are blessed in the schema, but IIRC the schema is open-ended (additional properties allowed) so you can add any that you need.

Reply:
PVA also defines the same set of limits:
- [`display.limitLow` and `display.limitHight`](https://docs.epics-controls.org/en/latest/pv-access/Normative-Types-Specification.html#display-t)
- [`control.limitLow` and `control.limitHight`](https://docs.epics-controls.org/en/latest/pv-access/Normative-Types-Specification.html#control-t)
- [`valueAlarm.lowWarningLimit`, `valueAlarm.lowAlarmLimit`, `valueAlarm.highWarningLimit`, `valueAlarm.highAlarmLimit`](https://docs.epics-controls.org/en/latest/pv-access/Normative-Types-Specification.html#alarmlimit-t)

The question here is what is useful downstream.

- If I am capturing a diode PV, do I want the display limits that would be used as axis ranges for plotting?
- If I am capturing a furnace setpoint PV, do I want the control limits that show I was close to the maximum temperature the device accepts?
- If I am capturing a thermocouple PV, do I want the warning or alarm limits to show that the sample was melting?

For that last point, we currently capture severity with each value, but we questioned the usefulness of that as maybe the scan should abort if the sample got too hot?

Reply:
| CA | PVA |
| -- | -- |
| lower_alarm_limit | valueAlarm.lowWarningLimit |
| upper_alarm_limit | valueAlarm.highWarningLimit |
| lower_ctrl_limit | control.limitLow |
| upper_ctrl_limit | control.limitHigh |
| lower_disp_limit | display.limitLow |
| upper_disp_limit | display.limitHigh |
| lower_warning_limit | valueAlarm.lowWarningLimit |
| upper_warning_limit | valueAlarm.highWarningLimit |

Presumably the correct equivalents?
Do we want to do some mapping to either/both to get something consistent?

Reply:
Yes, I think we should map both to the same, then add them to event model as optional.
However I only think we should do this for the useful fields...
@danielballan which fields do you think are useful?

Reply:
How about we have nested objects, so:
- `limits.control.low` and `limits.control.high`
- `limits.display.low` and `limits.display.high`
- `limits.warning.low` and `limits.warning.high`
- `limits.alarm.low` and `limits.alarm.high`